front/app/components/HookForm/FileUploader/index.tsx

```diff
@@ -11,7 +11,7 @@ import { get } from 'lodash-es';
 interface Props
   extends Omit<
     FileUploaderProps,
-    'onFileAdd' | 'onFileRemove' | 'files' | 'id'
+    'onFileAdd' | 'onFileRemove' | 'files' | 'id' | 'apiErrors'
```

Comment:
It's meant to prevent people from passing apiErrors manually, because the hook form component deals with the api error internally. So actually it can go. It's just a matter of having the error messages defined in the Error component's messages